convert_gvf_to_vcf/helpers.py

```diff
@@ -0,0 +1,25 @@
+# helpers.py is to prevent circular imports
```

Comment:
Why don't you put `generate_custom_structured_meta_line` in assistingconverter.py